Docs/KDESyntaxDefinition/dafny.xml

```diff
@@ -0,0 +1,151 @@
+      <item>yield</item>
+      <item>yields</item>
+    </list>
+    <list name="types">
+      <item>array</item>
+      <item>bool</item>
+      <item>char</item>
+      <item>imap</item>
+      <item>int</item>
+      <item>iset</item>
+      <item>map</item>
+      <item>multiset</item>
+      <item>nat</item>
+      <item>object</item>
+      <item>real</item>
+      <item>seq</item>
+      <item>set</item>
+      <item>string</item>
+    </list>
+
+    <list name="verification keywords">
+      <item>decreases</item>
+      <item>ensures</item>
+      <item>invariant</item>
+      <item>modifies</item>
+      <item>provides</item>
+      <item>reads</item>
+      <item>requires</item>
+      <item>reveals</item>
+    </list>
+
+    <contexts>
+      <context name="Normal" attribute="Normal Text" lineEndContext="#stay">
+        <keyword attribute="Keyword" String="keywords" context="#stay" />
+        <keyword attribute="Types" String="types" context="#stay" />
+        <keyword attribute="Assertions" String="verification keywords" context="#stay" />
+
+        <RegExpr attribute="Types" String="bv0|bv[1-9]\d*"/>
+        <RegExpr attribute="Types" String="array[2-9]\d*"/>
+
+        <Int attribute="Decimal" context="#stay" />
+        <Float attribute="Float" context="#stay" />
+        <HlCChar attribute="Char" context="#stay" />
+        <DetectChar attribute="String" char="&quot;" context="Quoted String" />
+
+        <Detect2Chars attribute="Comment" char="/" char1="/" context="SingleLineComment" />
+        <Detect2Chars attribute="Comment" char="/" char1="*" context="MultiLineComment" />
+      </context>
+
+      <context name="Quoted String" attribute="String" lineEndContext="#pop">
+        <Detect2Chars attribute="String" context="#stay" char="\\" char1="&quot;"/>
+        <DetectChar attribute="String" char="&quot;" context="#pop" />
+      </context>
+
+      <context name="SingleLineComment" attribute="Comment" lineEndContext="#pop"/>
+      <context name="MultiLineComment" attribute="Comment" lineEndContext="#stay">
+        <Detect2Chars attribute="Comment" context="#pop" char="*" char1="/"/>
+      </context>
+      <context name="Documentation" attribute="Comment" lineEndContext="#pop" />
+    </contexts>
+
+    <itemDatas>
+      <itemData name="Normal Text" defStyleNum="dsNormal" />
+
+      <itemData name="Keyword" defStyleNum="dsKeyword" />
+      <itemData name="Types" defStyleNum="dsDataType" />
+      <itemData name="Assertions" defStyleNum="dsOthers" />
+
+      <itemData name="Decimal" defStyleNum="dsDecVal" />
+      <itemData name="Float" defStyleNum="dsFloat" />
+      <itemData name="Char" defStyleNum="dsChar" />
+      <itemData name="String" defStyleNum="dsString" />
+
+      <itemData name="Comment" defStyleNum="dsComment" />
+    </itemDatas>
+  </highlighting>
+
+  <general>
+    <comments>
+      <comment name="singleLine" start="//" />
+      <comment name="multiLine" start="/*" end="*/" />
```

Comment:
Does this handle nested comments?

Reply:
The merged version handles nested comments.